README.md

```diff
@@ -28,19 +28,86 @@ colcon build
 source install/local_setup.bash
 ~~~
 
-## Available Parameters
-Format: `param name` type - description (default)
-
-- `gscam_config` string - with the stream config
-- `sync_sink` bool - sync wiht the clock (true)
-- `preroll` bool - prefill buffers (false)
-- `use_gst_timestamps` bool - Use gst time instead of ROS time (false)
-- `image_encoding`, string - encoding type (`sensor_msgs::image_encodings::RGB8`)
-- `camera_info_url`, string - Location of camera info file
-- `camera_name` string - Camera name 
-- `frame_id` string - Camera frame id (camera_frame)
+## Usage
+
+Make sure your GStreamer pipeline runs successfully in gst-launch-1.0.
+For example, here's a pipeline that works for the [Blue Robotics HD USB Camera](https://bluerobotics.com/store/sensors-sonars-cameras/cameras/cam-usb-low-light-r1/):
+~~~
+gst-launch-1.0 -v v4l2src device=/dev/video1 do-timestamp=true ! queue ! video/x-h264,width=1920,height=1080,framerate=30/1 ! h264parse ! queue ! avdec_h264 ! autovideosink
+~~~
+
+Here's the same pipeline in gscam2:
+~~~
+export GSCAM_CONFIG="v4l2src device=/dev/video1 do-timestamp=true ! queue ! video/x-h264,width=1920,height=1080,framerate=30/1 ! h264parse ! avdec_h264 ! videoconvert"
+ros2 run gscam gscam_main
+~~~
+
+Here's an example with parameters:
+~~~
+ros2 run gscam gscam_main --ros-args --remap /image_raw:=/my_camera/image_raw --params-file gscam_params.yaml -p camera_info_url:=file://$PWD/my_camera_info.ini
+~~~
+... where gscam_params.yaml is:
+~~~
+/gscam_publisher:
```

Comment:
Are you sure that you include the `/` ? I thought it was just the node name

Reply:
Good catch, thanks. Both work, as it turns out, but the unqualified version is the one documented by ROS2, so let's go with that one.